worker/caasunitprovisioner/application_worker.go

```diff
@@ -189,6 +202,20 @@ func (aw *applicationWorker) loop() error {
 					return errors.Trace(err)
 				}
 			}
+		case _, ok := <-appOperatorWatcher.Changes():
+			if !ok {
```

Comment:
Need to do the same processing as for unit changes, ie
debug message
if ! ok stop watcher and continue etc